source/includes/v2/_odsql.md

```diff
@@ -201,7 +201,7 @@ List of reserved keywords:
 
 The select clause can be used in records search APIs as the parameter `select`.
 
-The select clause allows to:
+The select clause allows one to:
```

Comment:
The select clause allows : 
- choosing the fields that will be returned for each row
- transforming fields using arithmetic
- renaming fields